dbt_checkpoint/check_model_parents_schema.py

```diff
@@ -71,6 +79,15 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         help="Blacklisted schemas.",
     )
 
+    parser.add_argument(
```

Comment:
@kiliantscherny why not using Python's [argparse choices](https://docs.python.org/3/library/argparse.html#choices)?

Reply:
Thanks for the suggestion, I've implemented the 2 choices of `""` and `"config"` 👍